docs/functions-suggestion-handlers.md

```diff
@@ -0,0 +1,218 @@
+## Block Kit Suggestion Handlers
+
+Your application's [functions][functions] can do a wide variety of interesting
+things: post messages, create channels, or anything available to developers via
+the [Slack API][api]. One of the more compelling features available to app developers
+is the ability to use [Block Kit][block-kit] to add richness and depth to messages
+in Slack. Even better, [Block Kit][block-kit] supports a variety of [interactive components][interactivity]!
```

Comment:
:twocents: I was wondering about built-ins while reading this (normally wouldn't call the Slack API unless I couldn't use a built-in)

`Your application can use our built-in functions to interact with the Slack API, or you can call our endpoints directly within your custom functions. One reason you might to do that is to add extra formatting and interactivity to messages sent through your app. [Does Schema.slack.functions.SendMessage support displaying messages with Block Kit as well?]`

Reply:
I think the sendmessage built-in does indeed support block kit payloads: https://github.com/slackapi/deno-slack-sdk/blob/main/src/schema/slack/functions/send_message.ts#L42

Reply:
(Unrelated to this PR but) Found [this](https://slack-pde.slack.com/archives/C01FA1FMGE9/p1663719958337829?thread_ts=1663649516.729709&cid=C01FA1FMGE9) helpful thread; Rich Text block or text accepted in the SendMessage message property and button type blocks accepted in the interactive_blocks property. No wonder the section block I was trying to pass kept erroring out